RevenueCatUI/Templates/V2/Components/Image/ImageComponentView.swift

```diff
@@ -46,26 +48,36 @@ struct ImageComponentView: View {
             )
         ) { style in
             if style.visible {
-                RemoteImage(
-                    url: style.url,
-                    lowResUrl: style.lowResUrl,
-                    darkUrl: style.darkUrl,
-                    darkLowResUrl: style.darkLowResUrl
-                ) { (image, size) in
-                    self.renderImage(image, size, with: style)
+                VStack(spacing: 0) {
```

Comment:
I'm curious: was it necessary to add the `VStack`? Didn't the `onWidthChange` work reliably when done directly on `RemoteImage`?

Reply:
Oh yeah! I did it at first because I wanted to 100% separate things but I meant to remove it 😅 Will try!